crawler/file_handler.py

```
@@ -246,6 +187,65 @@ def _extract_hwpx(path: Path) -> tuple[str, str]:
         return "", "hwpx_xml"
 
 
+def _xml_local_name(tag: str) -> str:
+    return tag.rsplit("}", 1)[-1] if "}" in tag else tag
+
+
+def _extract_docx_xml_text(raw_xml: bytes) -> str:
+    root = ET.fromstring(raw_xml)
```

Ruff (0.15.12)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)